src/utils/authcore.ts

```diff
@@ -29,7 +30,8 @@ export async function initAuthcore(
   }: {
     accessToken?: string;
     containerId?: string;
-  } = {}
+  } = {},
+  authcoreInitialScreen: AuthCoreInitialScreen = 'signin'
```

Comment:
I think we can put this in the object params
```suggestion
    initialScreen?: AuthCoreInitialScreen;
  } = {},
```

Reply:
[🎨 Improve params format](https://github.com/likecoin/likecoin-wallet-connector/pull/61/commits/3ece5dde6af985c0352f815d9a4487d50c9bfcf3)